prebid-server/developers/add-new-bidder-go.md

```diff
@@ -918,6 +918,21 @@ You need to provide default settings for your bid adapter. You can decide if you
 {: .alert.alert-warning :}
 **HOST SPECIFIC INFO:** The default endpoint must not be specific to any particular host, such as Xandr/AppNexus. We may ask you about suspicious looking ids during the review process. Please reach out to individual hosts if you need to set specialized configuration.
 
+## Aliasing an Adapter
+
+If your bidding endpoint can support more than one biddercode, you shouldn't replicate
+the whole adapter codebase. Rather, follow these steps to create a 'hardcoded' alias:
+
+1. Create a config yaml file in static/bidder-info - e.g. static/bidder-info/myalias.yaml
```

Comment:
@bretg Did you mean to have these all labeled as #1? Or should these steps be 1,2,3,4,5?

Reply:
When jekyll is converted to HTML, it should create an ordered list of this syntax. Setting all lines to 1 is nice in case we want to add one in the middle later on, no need to renumber everything below.